Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob.csproj

```diff
@@ -22,12 +22,12 @@
   </ItemGroup>
```

Comment:
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/a74fe5cf25a9d9d081dc89cd93cbc3c5f46a5ea9/Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob.csproj#L5
Please increase version to 1.0.1.